packages/system/src/border/border.test.ts

```diff
@@ -22,19 +23,6 @@ import {
   borderBottomRightRadius,
 } from './border';
 
-const borders = [0, 1, 2];
-
-[borders.zero, borders.small, borders.medium] = borders;
-
-const radii = [0, 4, 8, 9999];
-
-[radii.sharp, radii.semiRounded, radii.rounded, radii.circle] = radii;
-
-const colors = {
-  vibin: '#D8385E',
-  hope: '#1D856C',
-};
-
```

Comment:
You can use an object to create mock data instead of using tokens, this way you can make the test simpler :)

Reply:
Yes! I'll adjust it!

Thanks for the suggestion 🚀